### ISSUE TITLE
Activity ID = Video URL ?

### ISSUE THREAD

**Issue by sfraysse** [jhaag75/xapi-videojs#26]
Currently, the activity ID is the video URL (e.g. `http://vjs.zencdn.net/v/oceans.mp4`).

This solution may be reliable enought if the video URL is unique and stable. However, in some cases, the video URL may change (relocation, alternative formats, etc) and this is a problem in terms of data analysis.

So could we make a difference between the video source (an URL that may change) and the activity ID (a stable IRI)?

In other words, could be define the activity ID in the same way that we define the activity title, with a variable?

**Comment by jhaag75** [comment]
Yes, you could change it to anything you like for your activity. We just used this particular video for this demo. But you could update your implementation to not make the activity ID the video URL.

**Comment by liveaspankaj** [comment]
+1

Regards,
Pankaj Agrawal
